src/Runtime.Base/src/System/Runtime/StackFrameIterator.cs

```diff
@@ -13,6 +13,117 @@ internal unsafe struct REGDISPLAY
         internal UIntPtr SP;
     }
 
+    internal unsafe struct EHClauseIterator
```

Comment:
Could you please put this into a sepeate .cs file?

Reply:
Done